autoload/SpaceVim/layers/github.vim

```diff
@@ -1,11 +1,83 @@
+"=============================================================================
+" github.vim --- github layer file for SpaceVim
+" Copyright (c) 2018 Shidong Wang & Contributors
+" Author: Shidong Wang < wsdjeg at 163.com >
+" URL: https://spacevim.org
+" License: MIT license
+"=============================================================================
+
+""
+" @section github, layer-github
+" @parentsection layers
+" This layer provides GitHub integration for SpaceVim
+"
+" @subsection Mappings
+" >
+"   Mode      Key           Function
+"   -------------------------------------------------------------
+"   normal    SPC g h i     show issues
+"   normal    SPC g h a     show activities
+"   normal    SPC g h d     show dashboard
+"   normal    SPC g h f     show current file in browser
+"   normal    SPC g h I     show issues in browser
+"   normal    SPC g h p     show PRs in browser
+" <
+
 function! SpaceVim#layers#github#plugins() abort
-    return [
-                \ ['junegunn/vim-github-dashboard',      { 'on_cmd':['GHD','GHA','GHActivity','GHDashboard']}],
-                \ ['jaxbot/github-issues.vim',               { 'on_cmd' : 'Gissues'}],
-                \ ['wsdjeg/GitHub-api.vim'],
-                \ ]
+  return [
+        \ ['jaxbot/github-issues.vim', { 'on_cmd' : 'Gissues' }],
+        \ ['junegunn/vim-github-dashboard', {
+        \ 'on_cmd': ['GHActivity', 'GHDashboard'],
+        \ }],
+        \ ['rhysd/github-complete.vim', {
+        \ 'on_ft': ['gitcommit', 'markdown'],
```

Comment:
for gitcommit, we have build-in complete function, I do not want to use other plugin for it, but I think it can be enabled in markdown.

Reply:
You mean github-issues.vim? github-complete.vim provides more completion like emoji, username, and issue no, so it looks better to me.
Is there any other reason we should choose github-issues.vim?

Reply:
@sei40kr no, I mean complete issue and pr after `#`,  this is not github-issue.vim, it is build-in plugin,  

https://github.com/wsdjeg/SpaceVim/blob/replace/autoload/SpaceVim/plugins/gitcommit.vim

Reply:
for emoji, I think deoplete support multiple source, we can find out an emoji source.

Reply:
using rplugin source will run completion async, if we use omni func, it is not async, it will block your vim before the popmenu is opened.

in deoplete, all source are running async, but only the omni source is not. if you need emoji completion, maybi you can have a try with https://github.com/fszymanski/deoplete-emoji

Reply:
@wsdjeg 
I see. I'm going to fix this PR.

Reply:
I considered using github-complete.vim in markdown files, but I decided to add them (emoji/username completions) another time. I'll search ways to include them as asynchronous completions.

Reply:
ok,  :)  but for commit message, I want to use the build-in plugin. and I will add async  and cache support.